pkgs/development/libraries/libucl/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, pkgconfig, automake, autoconf, libtool }:
+
+let
+  pname = "libucl";
+  version = "0.8.1";
+in
+
+stdenv.mkDerivation {
+  inherit pname version;
```

Comment:
I've explicitly not used rec here, because I find that while it is convenient to use rec, it is more explicit to *know* which attributes are reused. What do you think?

Reply:
Using `rec`s makes the code just a bit more readable, and not using it, makes it harder / impossible for automatic updates such as [`nix-update`](https://github.com/Mic92/nix-update/#known-bugs) and https://github.com/ryantm/nixpkgs-update to do their job.

Reply:
> not using it, makes it harder / impossible for automatic updates such as nix-update and https://github.com/ryantm/nixpkgs-update to do their job

Good point, I wasn't aware these tools had difficulty with that syntax. Alright, I'll use rec.